ckanext/benap/benap_schema.json

```diff
@@ -321,6 +321,48 @@
       "form_snippet": null,
       "validators": "phone_number_validator benap_contact_point_org_fields_consistency_check"
     },
+    {
+      "field_name": "publisher",
+      "label":{
+        "en": "Publisher",
+        "nl":"Uitgever",
+        "de": "Verleger",
+        "fr":"Editeur"
+      },
+     "help_text": {
+        "en": "The \"publisher\" describes an entity (company and person) that publishes the datasets (we are not refering to the metadata here, but rather to the data themselves). He or she is responsible for the given information and must be contacted by data consumers if a contract needs to be concluded.",
+        "fr": "L'\"éditeur\" décrit une entité (entreprise et personne) qui publie les jeux de données (nous ne nous référons pas ici aux métadonnées mais plutôt aux données elles-mêmes). Il ou elle est responsable des informations données et doit être contacté(e) par les consommateurs de données si un contrat doit être conclu.",
+        "nl": "De \"uitgever\" beschrijft een entiteit (bedrijf of persoon) die de datasets publiceert (we verwijzen hier niet naar de metadata maar eerder naar de gegevens zelfs). Hij of zij is verantwoordelijk voor de gegeven informatie en moet door gegevensverbruikers gecontacteerd worden indien een overeenkomst moet worden gesloten.",
+        "de": "Der \"Herausgeber\" beschreibt eine Entität (Unternehmen oder Person), die die Datensätze publiziert (wir beziehen uns hier nicht auf die Metadaten sondern eher auf die Daten selbst). Er oder sie ist verantwortlich für die gegebene Information und muss von Datenverbrauchern kontaktiert werden, wenn ein Vertrag geschlossen werden muss."
+      },
+      "form_snippet": "nested_field.html",
+      "nested_fields": [
```

Comment:
enkel first and surname in de nested fields. andere velden worden automatisch ingevuld dus mogen niet getoond worden in de form of in de nested fields.